integrations/cache/cache-compat.php

```diff
@@ -70,7 +72,16 @@ public function add_cookie_script() {
 
 		$type_attr = current_theme_supports( 'html5', 'script' ) ? '' : ' type="text/javascript"';
 
-		echo "<script{$type_attr}>\n{$js}\n</script>\n"; // phpcs:ignore WordPress.Security.EscapeOutput
+		$script = "<script{$type_attr}>\n{$js}\n</script>\n";
+
+		/**
+		 * Filters the content of the script tag that creates the cookie.
```

Comment:
```suggestion
		 * Filters the script tag that creates the cookie.
```
Not only the content, any attributes attached to the tag as well.